test-js/unit/pentaho/visual/base/View.Spec.js

```diff
@@ -0,0 +1,67 @@
+define([
+  "pentaho/visual/base/View",
+  "pentaho/visual/base",
+  "pentaho/type/Context"
+], function(View, modelFactory, Context) {
+  "use strict";
+
+  describe("pentaho/visual/base/View", function() {
+    var element, dataSpec, Model, model;
+
+    beforeEach(function() {
+      element = document.createElement("div");
+      dataSpec = {
+        width: 1,
+        height: 1,
+        data: {v: {}}
+      };
+      Model = modelFactory(new Context());
```

Comment:
No, the new formulation looks good (besides, the sync/async hybrid formulation you're proposing does not really work, if I read it correctly :-P ). 
You don't need to use the `getAsync` method because the `get` method will succeed, because when given a type factory, like `modelFactory`, there's no need to load any other AMD modules.